lambdas/pkgpush/src/t4_lambda_pkgpush/__init__.py

```diff
@@ -4,197 +4,90 @@
-    'id': 'https://quiltdata.com/package-revision-location/1',
-    'type': 'object',
-    'properties': {
-        **PACKAGE_ID_PROPS,
-        'top_hash': {
-            'type': 'string',
-            'pattern': '^[0-9a-f]{64}$',
-        }
-    },
-    'required': [
-        *PACKAGE_ID_PROPS,
-        'top_hash',
-    ],
-}
-
-PACKAGE_BUILD_META_PROPS = {
-    'message': {
-        'type': 'string',
-    },
-    'meta': {
-        'type': 'object',
-    },
-    'workflow': {
-        'type': ['string', 'null']
-    },
-}
-
-PACKAGE_PROMOTE_SCHEMA = {
-    '$schema': 'http://json-schema.org/draft-07/schema#',
-    'type': 'object',
-    'properties': {
-        'parent': PACKAGE_REV_LOCATION_SCHEMA,
-        **PACKAGE_ID_PROPS,
-        **PACKAGE_BUILD_META_PROPS,
-    },
-    'required': [
-        'parent',
-        *PACKAGE_ID_PROPS,
-    ],
-}
-
-PACKAGE_LOCATION_SCHEMA = {
-    '$schema': 'http://json-schema.org/draft-07/schema#',
-    'id': 'https://quiltdata.com/package-location/1',
-    'type': 'object',
-    'properties': {
-        **PACKAGE_ID_PROPS,
-    },
-    'required': list(PACKAGE_ID_PROPS),
-    'additionalProperties': False,
-}
-
-PKG_FROM_FOLDER_SCHEMA = {
-    '$schema': 'http://json-schema.org/draft-07/schema#',
-    'type': 'object',
-    'properties': {
-        'registry': {
-            'type': 'string',
-        },
-        'entries': {
-            'type': 'array',
-            'items': {
-                'type': 'object',
-                'properties': {
-                    'logical_key': {'type': 'string'},
-                    'path': {'type': 'string'},
-                    'is_dir': {'type': 'boolean'},
-                },
-                'required': ['logical_key', 'path', 'is_dir'],
-            },
-        },
-        'dst': PACKAGE_LOCATION_SCHEMA,
-        **PACKAGE_BUILD_META_PROPS,
-    },
-    'required': [
-        'registry',
-        'entries',
-        'dst',
-    ],
-    'additionalProperties': False,
-}
-
-
-PACKAGE_CREATE_SCHEMA = {
-    'type': 'object',
-    'properties': {
-        **PACKAGE_ID_PROPS,
-        **PACKAGE_BUILD_META_PROPS,
-    },
-    'required': PACKAGE_ID_PROPS,
-    'additionalProperties': False
-}
-
-
-PACKAGE_CREATE_ENTRY_SCHEMA = {
-    'type': 'object',
-    'properties': {
-        'logical_key': {
-            'type': 'string'
-        },
-        'physical_key': {
-            'type': 'string'
-        },
-        'size': {
-            'type': 'integer'
-        },
-        'hash': {
-            'type': 'string'
-        },
-        'meta': {
-            'type': 'object',
-        },
-    },
-    'required': ['logical_key', 'physical_key'],
-}
-
-
-s3 = boto3.client('s3')
-lambda_ = boto3.client('lambda', config=botocore.client.Config(read_timeout=S3_HASH_LAMBDA_READ_TIMEOUT))
+S3_HASH_LAMBDA_CONCURRENCY = int(os.environ["S3_HASH_LAMBDA_CONCURRENCY"])
+S3_HASH_LAMBDA_MAX_FILE_SIZE_BYTES = int(
+    os.environ["S3_HASH_LAMBDA_MAX_FILE_SIZE_BYTES"]
+)
```

Comment:
how about using 119 line length with black?

Reply:
sure, we just need to configure that somewhere